pkg/apis/serving/v1beta1/trained_model.go

```diff
@@ -0,0 +1,44 @@
+package v1beta1
```

Comment:
Should this new API live under
apis/trainedmodels/v1beta1/...

I know the names are a bit odd right now, but I'd like to see:
```
apis/trainedmodel/v1beta1/...
apis/inferenceservice/v1alpha2/...
apis/inferenceservice/v1beta1/...
apis/inferencerouter/v1beta1/...
```

All of these should be owned by the same group "serving.kubeflow.org", so you may need to factor out register.go up into 
```
apis/register.go
```

Feel free to cut an issue to clean this up afterwards.

Reply:
I submitted git issue: https://github.com/kubeflow/kfserving/issues/1012. You want to do it so we can release different apis independently right?